### ISSUE TITLE
Agents probed successfully but no devices reported as found  

### ISSUE THREAD

**Issue by ltomuta** [TrakHound/TrakHound-Community#45]
![image](https://user-images.githubusercontent.com/2544090/194533758-e81b78be-75de-467d-87d0-35f1c664b5db.png)

**Comment by PatrickRitchie** [comment]
@ltomuta This was something that was supposed to be addressed in the latest release v1.5.9. Can you manually add the device you are looking for? I know there was an issue in the previous version that wouldn't read using the Auto Detect page but would still let you add the device manually.

**Comment by ltomuta** [comment]
Somehow I think I've only been able to add them manually in 1.2.17 (I accidentally forked your old instance of this repo so I ended up building that first :) ) Now I have the 1.5.9 built and ready to debug but our machines are down over the weekend.

**Comment by ltomuta** [comment]
Just verified, manually adding the device fails as well. The URI built for accessing it is missing the "/probe" part and the machine responds with ` <Error errorCode="UNSUPPORTED">The following path is invalid: /Machine</Error>`

**Comment by ltomuta** [comment]
Adding "/probe" to the uri in Probe's ExecuteAsync solves the issue:

![image](https://user-images.githubusercontent.com/2544090/194810629-8e8b072f-02d9-466d-88f3-ede169645921.png)

**Comment by PatrickRitchie** [comment]
@ltomuta Yes it looks like you are correct as the ExecuteAsync() should use the code used in Execute() to build the URL:

#### ExecuteAsync(): Incorrect
![image](https://user-images.githubusercontent.com/13772118/195877039-52965d85-9bd7-493d-9c14-3dced308954c.png)

#### Execute(): Correct
![image](https://user-images.githubusercontent.com/13772118/195877312-c3361869-a2db-4449-9fe3-6df0a66f9041.png)

This was a bug I had previously found as adding a DeviceName to the URL removes the /probe using the Uri() constructor. I added a new CombineUrl() method that I use in my newer applications to resolve this issue but apparently I only changed it in the Execute() method.

I will publish a new release that includes this update this weekend.

Thanks,
-Patrick
